src/bindings/python/BUILDING.md

```diff
@@ -1,88 +1,145 @@
-# Building the nGraph Python* API
+# Building the OpenVINO™ Python* API
```

Comment:
why do we need dedicated instruction for python? we have Wiki for how to build OpenVINO for a lot of platforms, where python specific there is just to install dependnecies from req.txt files

Other python things are handled automatically:
- python libs etc are installed via install_build_dependencies.sh
- ENABLE_PYTHON is selected automatically if python libs are found
- ENABLE_WHEEL is now not selected automatically, but we have PR https://github.com/openvinotoolkit/openvino/pull/11278 to finalize for this.

So, to build OV and to build OV + python is the same. The is a single req.txt file with cython version (BTW, why not to install cython via apt-get? it will simplify life for Linux users at least)

Reply:
> BTW, why not to install cython via apt-get? it will simplify life for Linux users at least

Does it? It install it in system globally. When using `pip` it is restricted to current Python environment. Both ways are possible, and second one is preferred from my experience. For some reason MacOS instructions mention it this way.

> I believe nobody will find this instruction

Bold assumption... It is located in the main folder responsible for component where potential developers will be looking for anything useful. I would advise to keep both Wiki and this file in check.

> ENABLE_PYTHON is selected automatically if python libs are found

Sure, I did not mentioned that it is off. Just pointed what to set.

![image](https://user-images.githubusercontent.com/65762745/184223568-5464d271-95a3-4cb3-8112-21e7207cc602.png)
About Wiki page and "everything being already written"... I will just leave it there.

One thing to notice, my instructions came first here and later wheel instructions were added:
![image](https://user-images.githubusercontent.com/65762745/184222760-3354f19c-b726-435a-9a2b-79f456cd7e56.png)
CC @adrianboguszewski just FYI that you are mentioned

All in all I would love to mention `ENABLE_WHEEL` if it is finalized. However keeping setup is also fine as many people are asking about it and use it currently.

Reply:
@jiwaszki I agree with Ilya, why do we need this dedicated introduction?
I would like to have only one place with instruction how to build OpenVINO.
At the current moment the build instruction is located on WIki and OpenVINO Readme has a reference to this place: https://github.com/openvinotoolkit/openvino#how-to-build

If each component which can be build separately (like MO, python or c APIs) will introduce new build instruction we will have a chaos, at the current moment we provide a common entry point for future OpenVINO developers. If something is changed in the build process, it is easier to change if it is located in one place.

Reply:
@ilyachur @ilya-lavrenov @jane-intel 

As discussed offline: this will be a first-step, as later it will be converged with Wiki pages and adjusted appropriately. Let's focus on this joint effort in near future! 😃